src/components/Form/Form.jsx

```diff
@@ -36,6 +38,11 @@ const Form = () => {
   const handleChangeCat = (event) => {
     setCategory(event.target.value)
   }
+
+  const handleChangeType = (event) => {
+    setType(event.target.value)
+  }
+
   const handleChangeLoc = (event) => {
     setWhereLost(event.target.value)
   }
```

Comment:
If the state is an object there won't be any need for this many functions to set the state!